packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -63,10 +63,7 @@ export default class Toolbar extends React.Component<Props> {
         const contentArray = Array.isArray(content) ? content : [content];
         // Remove empty/null/undefined elements, then map to Views
         return contentArray.filter((c) => c).map((content, i) => (
-            <View
-                style={[sharedStyles.content, sharedStyles.verticalAlign]}
-                key={i.toString()}
-            >
+            <View style={[sharedStyles.verticalAlign]} key={i.toString()}>
```

Comment:
This can be `style={sharedStyles.verticalAlign}` although I don't think this is necessary if you set `alignItems: "center"` on styles.container.  Then you can get rid of the style prop here.  Also, since there's no padding on the wrapper is the `filter` necessary?